src/UKHO.S100PermitService.API/Controllers/PermitController.cs

```diff
@@ -24,6 +24,21 @@ public PermitController(IHttpContextAccessor httpContextAccessor, ILogger<Permit
             _permitService = permitService ?? throw new ArgumentNullException(nameof(permitService));
         }
 
+        /// <summary>
```

Comment:
Should just make it clear that the Permit is supplied as a Zip-encoded stream, not a file as such

Reply:
Resolved